validator/validator.py

```diff
@@ -84,3 +84,34 @@ def validate(req, rules, return_errors=False):
         return result, errors
     # return validation result
     return result
+
+
+def validate_many(requests, rules, return_errors=False):
+    """
+       Validates many requests with given rules
+
+       Parameters:
+           requests (list): requests
+           rules (dict): rules
+           return_errors (bool): True/False according the necessity of returning error messages (False by default)
+
+       Returns:
+           result (bool): the result of the validation (if return_errors parameter was False)
+           OR
+           (result, error_messages): pair of the validation result and error messages object (if return_errors was True)
+       """
+
+    def get_validation_result(_results, _errors, _return_errors=return_errors):
+        return all(_results) if not _return_errors else (all(_results), _errors)
+
+    results, errors = [], []
+    for request in requests:
+        if return_errors:
+            result, _errors = validate(request, rules, return_errors)
+            errors.append(_errors)
+        else:
+            result = validate(request, rules)
+
+        results.append(result)
```

Comment:
I think we might had miscommunication. Everything is good but now this code is returning (array, array) first one is array of booleans and second one is for error messages. Can you please change it to (bool, array)?  if any one of given requests fails on validation result should be False, else True.

Reply:
`all()` functions makes `bool` from array of reults bools. See unit tests for proof.

Reply:
My bad, messed up with files :)